### PR TITLE
Deactivate sonar on travis

### PR DESCRIPTION
**Pull request by BridgeAR**
# Deactivate sonar on travis and update Node.js version used to test to v10.x

## Definition of Done

A task/pull request will not be considered to be complete until all these items can be checked off.

- [x] **All** requirements mentioned in the issue are implemented
- [x] Does match the Code of Conduct and the Contribution file
- [x] Task has its own **GitHub issue** (something it is solving)
  - [x] Issue number is **included in the commit messages** for traceability
- [ ] **Update the README.md** with any changes/additions made
- [ ] **Update the CHANGELOG.md** with any changes/additions made
- [ ] **Enough test coverage to ensure that uncovered changes do not break functionality**
- [x] **All tests pass**
- [x] **Descriptive pull request text**, answering:
  - What problem/issue are you fixing?
  - What does this PR implement and how?
- [ ] **Assign your PR to someone** for a code review
  - This person _will be contacted **first**_ if a bug is introduced into `master`
- [x] **Manual testing** did not fail

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -136,7 +136,7 @@ jobs:
...
 stages:
   - test
-  - name: sonar
-    if: branch = master AND type != pull_request
-  - name: deploy
-    if: tag IS present
+  # - name: sonar
+  #   if: branch = master AND type != pull_request
+  # - name: deploy
+  #   if: tag IS present
```
Comment (alschmut) [inline .travis.yml:142]: @BridgeAR Mentioning a stage inside the stages only ensures the stages will run in order from top to bottom and each stage only runs if the conditions evaluate to true. So the `sonar` and `deploy` stages still run when commented out.

What you actually want is something like this, which only deactivates the sonar branch.
```
stages:
  - test
  - name: sonar
    if: false AND branch = master AND type != pull_request
  - name: deploy
    if: tag IS present
```

**Comment by NearW** [comment]
I updated the node travis version to 12 in another commit, which is already pushed to master.
We needed at least version 10 for the new puppeteer update, but version 12 is working as well.

**Comment by BridgeAR** [comment]
@NearW it would be great to test multiple Node.js versions (at least the LTS versions). And stopping support for the older Node.js version should be considered a breaking change. This is probably not a big concern right now due to the limited user scope but we might want to discuss our semver handling again.

**Comment by NearW** [comment]
> @NearW it would be great to test multiple Node.js versions (at least the LTS versions). And stopping support for the older Node.js version should be considered a breaking change. This is probably not a big concern right now due to the limited user scope but we might want to discuss our semver handling again.

That is a very good point. We should extend our CI by at least checking our lowest and highest suported node version. We could use the highest in general and when we do a release, we could run the pipelines with both versions. This would only slow down our deploy-pipeline.
However I'm not sure if this is technically possible. We should discuss this tomorrow.

**Pull request merged**